.cirrus.yml

```diff
@@ -157,6 +169,232 @@ task:
+    image_project: pg-ci-images-dev
+    image: family/pg-ci-netbsd-9-2
+    platform: netbsd
+    cpu: $CPUS
+    memory: 4G
+    disk: 25
+
+  sysinfo_script: |
+    locale
+    id
+    uname -a
+    ulimit -a -H && ulimit -a -S
+    export
+
+  ccache_cache:
+    folder: $CCACHE_DIR
+
+  create_user_script: |
+    useradd postgres
+    chown -R postgres:users /home/postgres
+    mkdir -p ${CCACHE_DIR}
+    chown -R postgres:users ${CCACHE_DIR}
+
+
+task:
+  <<: *netbsd_template
+
+  name: NetBSD - 13 - Autoconf
+  trigger_type: manual
+
+  # --with-uuid=bsd is not set since 'bsd' uuid option
+  # is not working on netBSD. See
+  # https://www.postgresql.org/message-id/17358-89806e7420797025@postgresql.org
+  # And other uuid options are not available on netBSD
+  configure_script: |
+    su postgres <<-EOF
+      ./configure \
+        --enable-cassert --enable-debug --enable-tap-tests \
+        --enable-nls \
+        \
+        ${BSD_CONFIGURE_FEATURES} \
+        \
+        --with-includes=/usr/pkg/include --with-libs=/usr/pgk/lib \
+        CC="ccache cc" \
+        CXX="ccache c++" \
+        CFLAGS="-Og -ggdb" \
+        \
+        PYTHON=python3.8
+    EOF
+  build_script: su postgres -c "gmake -s -j${BUILD_JOBS} world-bin"
+  upload_caches: ccache
+
+  test_world_script: |
+    su postgres <<-EOF
+      ulimit -c unlimited; \
+      gmake -s ${CHECK} ${CHECKFLAGS} -j${TEST_JOBS}
+    EOF
+
+  on_failure:
+    <<: *on_failure
+
+
+task:
+  <<: *netbsd_template
+
+  name: NetBSD - 13 - Meson
+
+  # -Duuid=bsd is not set since 'bsd' uuid option
+  # is not working on netBSD. See
+  # https://www.postgresql.org/message-id/17358-89806e7420797025@postgresql.org
+  # And other uuid options are not available on netBSD
+  configure_script:
+    - su postgres -c 'meson setup --buildtype debug -Dcassert=true -Dssl=openssl build'
+
+  build_script: su postgres -c 'ninja -C build'
+  upload_caches: ccache
+
+  test_world_script:
+    - su postgres -c 'meson test --no-rebuild -C build'
+
+  always:
+    meson_log_artifacts:
+      path: "build/meson-logs/*.txt"
+      type: text/plain
+
+  on_failure:
+    <<: *on_failure
+
+
+openbsd_template: &openbsd_template
+  env:
+    CPUS: 2
+    BUILD_JOBS: 3
+    TEST_JOBS: 3
+
+    CIRRUS_WORKING_DIR: /home/postgres/postgres
+    CCACHE_DIR: /tmp/ccache_dir
+
+    PATH: /usr/sbin:$PATH
+    LD_LIBRARY_PATH: $LD_LIBRARY_PATH:/usr/local/lib
+    TCL_CONFIG_PATH: /usr/local/lib/tcl/tcl8.6/
+
+    # Postgres interprets LANG as a 'en_US.UTF-8' but it is 'C', then
+    # Postgres tries to set 'LC_COLLATE' to 'en_US.UTF-8' but it is not
+    # changeable. Initdb fails because of that. So, LANG is forced to be 'C'.
+    LANG: "C"
+    LC_ALL : "C"
+
+    BSD_CONFIGURE_FEATURES: *BSD_CONFIGURE_FEATURES
+
+  only_if: $CIRRUS_CHANGE_MESSAGE !=~ '.*\nci-os-only:.*' || $CIRRUS_CHANGE_MESSAGE =~ '.*\nci-os-only:[^\n]*openbsd.*'
+
+  compute_engine_instance:
+    image_project: pg-ci-images-dev
```

Comment:
Need to set that to `$IMAGE_PROJECT` after anarazel/pg-vm-images/pull/9 is merged.